### DISCUSSION TITLE
Basic question about NETWORK variable

### DISCUSSION THREAD

**Discussion by schklom** [bubuntux/nordvpn#172] [Q&A]
Hi,

I need to be able to access this nordvpn container from my phone while outside.
I have set up port forwarding on my home router, but the setup only works when I input my phone's IP address in the `NETWORK` variable (`20.30.40.50/32` for example).

My phone obviously doesn't have a fixed IP address, so I think I need to set `NETWORK=0.0.0.0/0` to allow replies to any IP.

## Question
Is there any risk setting `NETWORK=0.0.0.0/0` ?
Can people find my server's IP address because of it ?

**Comment by moritzchow** [comment]
@schklom I do think the packets when sent to your nordvpn container they're allowed but then when they're routed to the wireguard server they'll be dropped if the packets are not recognized by the wireguard server with proper key so my guess is it's not really risky unless you have your key exposed to others.
BTW I can get the it work through inputting my phone's currrent ip address (for sure it's not a fixed one). It does not work re NETWORK=0.0.0.0/0 set in my nordvpn container tho (maybe it's due to that fact that I am using an unraid server to build both containers and I'm not using docker compose, can't be sure - I'm kinda new to this docker-container thing)

**Reply by schklom** [reply]
0.0.0.0/0 doesn't work for me either, and I'm not using unraid.
I'll make an issue about this, I think it comes from the container itself since it works with my phone's current ip address.
